File: programmers/Level1/CPP/문자열 내림차순으로 배치하기.cpp
#include <string>
using namespace std;

string solution(string s) {
    string answer = s;
    
    for(int i = 0; i < answer.size(); i++){
        int max = i;
        for(int j = i; j < answer.size(); j++){
            if(answer[max] < answer[j]){
                max = j;
            }
        }
        char c = answer[max];
        answer[max] = answer[i];
        answer[i] = c;
    }
    
    return answer;
}

File: programmers/Level1/JAVA/문자열 내림차순으로 배치하기.java
import java.util.Arrays; 

class Solution {
    public String solution(String s) {
        String answer = "";
        char[] chars = s.toCharArray();
        Arrays.sort(chars);
        answer = new String(chars);
        answer = new StringBuffer(answer).reverse().toString(); //문자열 뒤집기 함수
        return answer;
    }
}
